### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by weifuchow**
Signed-off-by: weifuchow <122757067@qq.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by weifuchow** [comment]
@Aisuko @leecalcote  good to go?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+[![Docker Pulls](https://img.shields.io/docker/pulls/layer5/meshsync.svg)](https://hub.docker.com/r/layer5/meshsync)
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/layer5io/meshsync)](https://goreportcard.com/report/github.com/layer5io/meshsync)
+
+[![Website](https://img.shields.io/website/https/layer5.io/meshery.svg)](https://layer5.io/meshery/)
+[![Twitter Follow](https://img.shields.io/twitter/follow/layer5.svg?label=Follow&style=social)](https://twitter.com/intent/follow?screen_name=mesheryio)
+[![Slack](http://slack.layer5.io/badge.svg)](http://slack.layer5.io)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3564/badge)](https://bestpractices.coreinfrastructure.org/projects/3564)
+
+# Service Mesh Synchronization
```
Comment (Aisuko) [inline README.md:12]: The title should be `MeshSync`?

Comment (leecalcote) [reply README.md:12]: Right, yes.

It is nice that "Service Mesh Synchronization" is in parenthesis below, but you're right, the name of this component is "MeshSync".

**Review by leecalcote — APPROVED** [review]
Nice!

**Review by leecalcote — APPROVED** [review]
You made quick work of this, @weifuchow. 💪

**Comment by leecalcote** [comment]
Yay! 🎈

**Comment by weifuchow** [comment]
@Aisuko @leecalcote Thanks for your help

**Pull request merged**